packages/mobile/src/screens/notifications-screen/content/AddTrackToPlaylist.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  AddTrackToPlaylist as AddTrackToPlaylistNotification,
+  Entity as EntityType
+} from 'audius-client/src/common/store/notifications/types'
+import { StyleSheet, Text, View } from 'react-native'
+
+import { useTheme } from 'app/utils/theme'
+
+import Entity from './Entity'
+import TwitterShare from './TwitterShare'
+import User from './User'
+
+const styles = StyleSheet.create({
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column'
+  },
+  text: {
+    fontFamily: 'AvenirNextLTPro-Bold',
```

Comment:
this is all fine, but check out some of the other files that use our style functions, looks kinda like this:

```
const useStyles = makeStyles(({typography}) => ({
  text: {fontFamily: typography.fontByWeight: 'bold'}
})
```
and then 
```
const Comp = () => {
  const styles = useStyles()
}
```

what we get out of this is the ability to add themable constants in the style object, like typography, palette, etc, lmk if you need more help on this, but the values that would prob work for text here are:

```
   fontFamily: typography.fontByWeight.bold,
   fontSize: typography.fontSize.medium,
   marginBottom: spacing(2)
```
where typography and spacing come come from the parameters of makeStyles like in example above